tests/pre_merge/models/test_feature_extractor.py

```diff
@@ -39,6 +42,32 @@ def test_feature_extraction(self, backbone, pretrained):
         else:
             pass
 
+    def test_torchfx_feature_extraction(self):
```

Comment:
Maybe add a test case for creating a feature extractor from a local model definition, and a string to a weights file. That way we would have a more complete coverage of the possible use cases of the class.

We could use a dummy model consisting of a few convolutional layers and corresponding weights file.

Reply:
I've added functionality to directly pass locally defined class + tests for it